### PR TITLE
14.0 mig crm phonecall

### PR DESCRIPTION
**Pull request by redapureskill**
**Migrating** crm_phonecall to **14.0** :+1:

### CODE REVIEW

**Comment by rousseldenis** [comment]
@redapureskill Please fix this runbot warning :

image

**Comment by redapureskill** [comment]
@rousseldenis should be okay now

**Comment by redapureskill** [comment]
@rousseldenis should be ok, and ready for review 🧐

**Comment by tonygalmiche** [comment]
I have this error on runbot : 
image

**Comment by redapureskill** [comment]
> I have this error on runbot :
> image

Fixed :+1: 
@tonygalmiche you can review it :)

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

image

**Review by HaraldPanten — APPROVED** [review]

**Review by dnplkndll — APPROVED** [review]
looks good, seems like most of the changes on crm_phonecall_report_view.xml on the mig commit should have been in the linting? 
Tested on my install script.

File: crm_phonecall/security/crm_security.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_scheduled_calls" model="res.groups">
```
Comment (atchuthan) [inline crm_phonecall/security/crm_security.xml:3]: This security group is not used in this module, so we could delete it if not required.

File: crm_phonecall/security/crm_security.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        <field name="name">Show Scheduled Calls Menu</field>
+        <field name="category_id" ref="base.module_category_hidden" />
+    </record>
+    <record id="group_show_form_view" model="res.groups">
```
Comment (atchuthan) [inline crm_phonecall/security/crm_security.xml:7]: Also, no view has this group defined.  Did I miss anything?

File: crm_phonecall/security/crm_security.xml (hunk 1 above)
Comment (redapureskill) [reply crm_phonecall/security/crm_security.xml:3]: it's is used in please check views/crm_phonecall_view.xml

File: crm_phonecall/security/crm_security.xml (hunk 2 above)
Comment (redapureskill) [reply crm_phonecall/security/crm_security.xml:7]: it's is used in please check views/crm_phonecall_view.xml

**Review by atchuthan — APPROVED** [review]
Looks good :+1:

**Review by atchuthan — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
LGTM. @redapureskill Do you want to keep commits as is or do you want to squash ?

**Comment by anajuaristi** [comment]
Something missing to be merged? 
Could we help with anything?

**Review by atchuthan — APPROVED** [review]
@rousseldenis  FYI, The commits are squashed already

**Comment by atchuthan** [comment]
Travis fails due to crm_lead_firstname module

**Comment by redapureskill** [comment]
> Travis fails due to crm_lead_firstname module

LGTM

**Pull request merged**